Includes/CubbyTower/Components/Targeter.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 CubbyTower Team
+// Chris Ohk, Minkyu Lee, Minjune Yi
+// We are making my contributions/submissions to this project solely in our
+// personal capacity and are not conveying any rights to any intellectual
+// property of any third parties.
+
+#ifndef CUBBYTOWER_TARGETER_HPP
+#define CUBBYTOWER_TARGETER_HPP
+
+namespace CubbyTower
+{
+//!
+//! \brief Targeter struct.
+//!
+//! This struct stores whether the tower/bullet can attack the target.
+//! 1 bit means attakable
+//! 100 : Air, 010 : ground, 001 : stealth
```

Comment:
how about 0b100?

Reply:
I prefer `1`, `1 << 1` and `1 << 2`.
Also, you should use separate variables to prevent magic numbers.